kamel-core/build.gradle.kts

```diff
@@ -174,4 +200,23 @@ tasks.withType<AbstractPublishToMaven>().configureEach {
     dependsOn(dependsOnTasks)
 }
 
+android {
+    namespace = "io.kamel.core.cache"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 21
+    }
+
+    buildFeatures {
+        compose = true
+        buildConfig = true

```

Comment:
This is true be default I think: https://developer.android.com/reference/tools/gradle-api/7.0/com/android/build/api/dsl/BuildFeatures#buildconfig

probably be removed?

Reply:
what's the point of enabling buildConfig if you don't create any configs anyway?

Reply:
TBH I forgot why I added build config, I remember adding it because I was accessing the "DEBUG" flag but it seems to be running fine without it.

Reply:
alright cool sounds good